.github/workflows/stale-issues.yml

```diff
@@ -0,0 +1,42 @@
+name: Check stale issues
+
+on:
+  workflow_dispatch: # FIXME: remove
+  schedule:
+    - cron: '0 12 * * *'
+
+env:
+  PERIOD: 14 # in days
+  WAIT_FOR_INFO: "waiting for more input"
+  DRY_RUN: true
+
+jobs:
+  check-issue-status:
+    name: Retrieve all open issues that are inactive
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Get inactive open issues
+        id: stale_issues
+        uses: ./.github/actions/stale_issues
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          period: ${{ env.PERIOD }}
+      - name: Comment on found issues
+        if: ${{ success() }}
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          for issue_url in ${{ steps.stale_issues.outputs.result }}; do
+            # Filter stale issues that were triaged and are not waiting for more information
+            stale_triaged=$(gh issue view $issue_url \
+                                    --json labels \
+                                    --jq '.labels | map(.name != "$WAIT_FOR_INFO") | all and length > 0')
+            if [[ $stale_triaged == 'true' ]]; then
+              if [[ $DRY_RUN == 'true' ]]; then
+                echo "Commenting on issue: $issue_url"
+              else
+                gh issue comment $issue_url --body "Hey, devs! I have been waiting for too long. Please give me some attention! :-)"
```

Comment:
It prints the URL to the comment. But notice that the command is: `gh issue comment ...` so moving it to the no dry-run scope will comment on the issue. If this is the case, we don't need a dry-run mode.

Reply:
I meant including the `echo` command in non-dry run.

If `gh issue` prints the link then I guess it's not necessary. But if you could make the `echo` print the issue title it would still be useful in both cases.